packages/scripts/config/webpack.config.js

```diff
@@ -89,18 +83,29 @@ const config = {
 	},
 	module: {
 		rules: [
-			{
-				test: /\.js$/,
-				use: require.resolve( 'source-map-loader' ),
-				enforce: 'pre',
-			},
 			{
 				test: /\.js$/,
 				exclude: /node_modules/,
-				use: {
-					loader: require.resolve( 'babel-loader' ),
-					options: getBabelLoaderOptions(),
-				},
+				use: [
+					'thread-loader',
```

Comment:
This needs to be:

```suggestion
					require.resolve( 'thread-loader' ),
```

Reply:
Oh, good catch. I forgot about it :(

Reply:
I was testing this with my test plugin but got de-railed to other things. Your comment made me realize I had this pending test to do :)

Reply:
Can we prevent it from happening in the future? Honestly speaking, I don't know how it could be approached.

Reply:
mmm, some thoughts:

- we could create a simple script that uses the default config and use its exit code to detect whether it had failed or not. Has Jest any utility to integrate this in its testing chain? Or perhaps it'd be a better idea to add it to Travis?

- a different approach would be: adding an unit-test that reads the default webpack config and checks that every loader is a valid path? If `require.resolve` is present, inspecting loaders should render something like `<path-to-gutenberg>/node_modules/thread-loader/...` instead of `thread-loader`?

Reply:
Some good thoughts @nosolosw, I don't have good answers here but it'd be great to have some automation in place on Travis.